### PR TITLE
Cleanup eslint

### PR DESCRIPTION
**Pull request by f-masche**

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -1,49 +1,40 @@
 {
   "extends": "airbnb",
```
Comment (haensl) [inline .eslintrc]: I actually was doing the exact same issue :D
But I came to the conclusion that `"extends": "eslint:recommended"` would get rid of some dependencies introduced by `airbnb` which we likely won't need anyway. Opinions?

File: .eslintrc [hunk 2]
```diff
@@ -1,49 +1,40 @@
...
-  "plugins": [
-    "import"
-  ],
   "env": {
-    "browser": true,
     "node": true,
-    "mocha": true
+    "mocha": true,
```
Comment (haensl) [inline .eslintrc]: No dangling comma please

File: .eslintrc [hunk 3]
```diff
@@ -1,49 +1,40 @@
...
+      "ecmaVersion": 6,
+      "sourceType": "script",
+      "impliedStrict": false
   },
   "rules": {
-    "react/require-extension": "off",
+    "strict": ["error", "global"],
     "consistent-return": "off",
-    "max-len": ["error", 200, 2],
+    "max-len": ["error", {
```
Comment (haensl) [inline .eslintrc]: Sweet! 👍

File: js-performance.js [hunk 4]
```diff
@@ -10,20 +11,19 @@ const DEFAULTS = require(join(__appRoot, 'support/defaults'));
...
 const clock = require(join(__appRoot, 'support/clock/clock'));
 const testdata = require(join(__appRoot, 'support/testdata/testdata'));
-
 const opts = new GetOpt([
-  ['h', 'help', 'Display this helptext.'],
-  ['i', 'iterations=', `Specify the number of iterations per profiled function. Default: ${DEFAULTS.iterations}.`],
-  ['q', 'quiet', 'Print results only.'],
-  ['m', 'magnitude=', `Specify the magnitude of testdata. Default: ${DEFAULTS.testdataMagnitude}.`],
-  ['v', 'verbose', 'Print verbose information.']
+    ['h', 'help', 'Display this helptext.'],
```
Comment (haensl) [inline js-performance.js]: That's four spaces, right? If so 👎

File: js-performance.js [hunk 5]
```diff
@@ -10,20 +11,19 @@ const DEFAULTS = require(join(__appRoot, 'support/defaults'));
...
+    ['v', 'verbose', 'Print verbose information.']
 ]).bindHelp()
-.parseSystem();
+  .parseSystem();
 
 let iterations = DEFAULTS.iterations;
-if ('iterations' in opts.options
-   && !isNaN(parseInt(opts.options.iterations, 10))) {
-     iterations = parseInt(opts.options.iterations, 10);
+if ('iterations' in opts.options &&
```
Comment (haensl) [inline js-performance.js]: Thought you were a proponent of the operator on the next line?

File: js-performance.js [hunk 6]
```diff
@@ -50,30 +50,31 @@ if (opts.argv.length > 0) {
   opts.argv.forEach((profileName) => {
     const discoveredProfiles = glob.sync(`profiles/**/@(${profileName}.profile|${profileName}.profile.js|${profileName}.js)`);
     if (discoveredProfiles.length === 1) {
-      profiles.push(require(join(__appRoot, discoveredProfiles.pop())));
-    } else if (speak){
+      profiles.push(require(join(__appRoot, discoveredProfiles.pop()))); // eslint-disable-line
```
Comment (haensl) [inline js-performance.js]: Let's rather tweak `.eslintrc` so we don't need these sorts of comments.

File: package.json [hunk 7]
```diff
@@ -21,8 +22,16 @@
   },
   "devDependencies": {
     "chai": "^3.5.0",
+    "eslint": "^3.15.0",
+    "eslint-config-airbnb": "^14.1.0",
```
Comment (haensl) [inline package.json]: If we move to `eslint:recommended` we can get rid of all of these modules we don't use anyway. I see neither `import` nor `jsx` or `react` on the horizon for this project. If - if they would ever find their way in here, we can easly add this and change the `.eslintrc` accordingly.

File: js-performance.js (hunk 4 above)
Comment (f-masche) [inline js-performance.js]: Wow, that was the eslint auto fixer

File: js-performance.js (hunk 5 above)
Comment (f-masche) [inline js-performance.js]: Eslints fault. I will adjust it.

File: package.json (hunk 7 above)
Comment (f-masche) [inline package.json]: Good idea!

File: .eslintrc (hunk 1 above)
Comment (f-masche) [inline .eslintrc]: Yes I thought of that myself. I just didn't want to change everything ;) Let's give it a try.

File: js-performance.js (hunk 4 above)
Comment (f-masche) [inline js-performance.js]: IMO this spacing is correct because its an array element here.

File: js-performance.js (hunk 5 above)
Comment (f-masche) [inline js-performance.js]: Done ca0f8be

File: package.json (hunk 7 above)
Comment (f-masche) [inline package.json]: Done ca0f8be

File: .eslintrc (hunk 2 above)
Comment (f-masche) [inline .eslintrc]: Done ca0f8be

File: .eslintrc (hunk 1 above)
Comment (f-masche) [inline .eslintrc]: Done ca0f8be

File: package.json [hunk 8]
```diff
@@ -7,7 +7,7 @@
     "start": "node js-performance.js",
     "test": "mocha spec.setup.js **/*.spec.js",
     "tdd": "mocha --recursive --watch spec.setup.js **/*.spec.js",
-    "lint": "eslint '**/*.js'"
+    "lint": "eslint --fix '**/*.js'"
```
Comment (haensl) [inline package.json]: Not sure if I like the auto-fix...actually I don't like it. One thing that burned into my mind from our common mentor is that you want to fail on errors until you fixed them. This is analogous to the cron-job-inode-issue. You remember?

File: js-performance.js (hunk 4 above)
Comment (haensl) [inline js-performance.js]: You are right.

**Pull request merged**